api/envoy/config/filter/fault/v2/fault.proto

```diff
@@ -22,9 +24,8 @@ message FaultDelay {
   // supported.
   FaultDelayType type = 1 [(validate.rules).enum.defined_only = true];
 
-  // An integer between 0-100 indicating the percentage of operations/connection requests
-  // on which the delay will be injected.
-  uint32 percent = 2 [(validate.rules).uint32.lte = 100];
+  // The percentage of operations/connection requests on which the delay will be injected.
+  envoy.type.FractionalPercent percent = 2;
```

Comment:
We can't just change this. You will need to deprecate the old percent and add a new one.

Reply:
call the new one as percentage. And i think there is also some protoc validation with it